db/seeds/development.rb

```diff
@@ -27,6 +27,12 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
+# Disable mail delivery for the duration of this task
+ActionMailer::Base.perform_deliveries = false
+
+# Avoid asynchronous DeliverWorkPackageCreatedJob
+Delayed::Worker.delay_jobs = false
```

Comment:
Everything is fine. Now that I understand it, I like it ^^

Actually I would expect that after seeding is done all things are completed and nothing is pending execution (e.g. repos). So this is actually a feature ^^